File: PPF/PPF/Model/Festival.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PPF.Model
{
    public class Festival
    {
        public int Id { get; set; }

        public string Naziv { get; set; }

        public string Forma { get; set; }

        public string Tema { get; set; }

        public string Sponzor { get; set; }

        public int BrPozorista { get; set; }

        public double Budzet { get; set; }

        public DateTime PocetakOdrzavanja { get; set; }

        public DateTime KrajOdrzavanja { get; set; }
    }
}

File: PPFUV_front/bp2/src/app/entities/festival/festival.ts
import { Forma } from '../forma/forma';
import { Pozoriste } from '../pozoriste/pozoriste';
import { Nagrada } from '../nagrada/nagrada';
import { Radnik } from '../radnik/radnik';
import { Izabran } from '../izabran/izabran';

export class Festival {
    id: number;
    naziv: string;
    tema: string;
    pocetakOdrzavanja: Date;
    krajOdrzavanja: Date;
    sponzor: string;
    brojPozorista: number; //* broj koji prima
    forma: Forma;
    budzet: number;
    pozoriste: Pozoriste;

    ucesnici: Array<Izabran>;
    
    nagrade: Array<Nagrada>;
    radnici: Array<Radnik>;
}
